Add images from hopper

urielsalis opened the pull request:
Adds images from gist
Name is hopper path minus index.html, in order of appeareance in the git

LordRalex commented:
Deploy preview for *youthful-colden-6bf88b* ready!

Built with commit 4f6d88213ffacc2a28de82b70d6e6bf2443f38db

https://deploy-preview-59--youthful-colden-6bf88b.netlify.com

LordRalex commented:
Would be good to also update the urls on the pages that use those so they work as-is

urielsalis commented:
Removed breadcrumbs(all broken and its already provided in the top), added missing images and fixed the link, next PR is a reformat after this is merged
